laterpay/asset_sources/js/laterpay-backend-account.js

```diff
@@ -62,32 +75,56 @@
             },
 
             autofocusEmptyInput = function() {
-                var $inputs = $('.lp_js_validateApiKey, .lp_js_validateMerchantId');
-                for (var i = 0, l = $inputs.length; i < l; i++) {
-                    if ($inputs.eq(i).val() === '') {
-                        $inputs.eq(i).focus();
+                var i = 0,
+                    l = $o.apiCredentialsInputs.length;
+
+                for (; i < l; i++) {
+                    if ($o.apiCredentialsInputs.eq(i).val() === '') {
+                        $o.apiCredentialsInputs.eq(i).focus();
                         return;
                     }
                 }
             },
 
+            toggleVisibilityInTestMode = function() {
+                if (hasNoValidCredentials()) {
+                    // FIXME: wtf is this?
+                    $o.testModeInvalidCredField.val(1);
```

Comment:
It's the way to notify backend about invalid credentials. By default this field 0, if credentials invalid - set to one.